data/sql/updates/pending_db_world/rev_1597618822007580900.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1597618822007580900');
+
+UPDATE `gameobject_template` SET `ScriptName`='go_midsummer_music' WHERE `entry`=188174;
```

Comment:
you don't need multiple SQL files, just 1 is enough. Also please add an empty line at the end of it ;)

Reply:
Okey, they are merged now, I also added the missing boxes to sattrath. To confirm is there a set guid I should have used for this, right now I have used the last free ones to add them based on a clean install.